packages/extension/src/pages/register/enable-chains/index.tsx

```diff
@@ -486,7 +486,7 @@ export const EnableChainsScene: FunctionComponent<{
 
         return new CoinPretty(
           chainStore.getChain(b.chainId).stakeCurrency,
-          "0"
+          "-1"
         );
       })();
```

Comment:
@renfengshi What's the intent of going from 0 to -1 here?

Reply:
To ensure the unsupported chains at bottom.

Reply:
@renfengshi I think you should keep this at 0 and do something different